### ISSUE TITLE
Password for binaries

### ISSUE THREAD

**Issue by craschnet** [fventuri/nrsc5#2]
Both nrsc5-windows32.zip and nrsc5-windows-64.zip are password protected ZIPs? 

Can I get the password. 

And why the ZIPs are password protected?

**Comment by HTTP-Requests** [comment]
my question exactly

**Comment by fanlessfan** [comment]
same question.

**Comment by robman501a** [comment]
Yeah, what's the password? How can you provide precompiled binaries that are password protected without providing a password? 
![image](https://github.com/user-attachments/assets/579c8ef6-356b-4c46-9115-69c195628dde)

**Comment by craschnet** [comment]
Again what's the password?

**Comment by tekphobia** [comment]
Found it. The password is literally this: accept all licenses
